docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination.md

```diff
@@ -27,7 +27,7 @@ To deploy LogGroup Lambda Connector, do the following:
 1. In the Serverless Application Repository, search for **sumologic**.
 1. Select **Show apps that create custom IAM roles or resource policies** check box.
 1. Click the **sumologic-loggroup-connector**,link, and then click **Deploy**.
-1. In the **AWS Lambda \> Functions \> Application Settings** panel, configure the parameters as described in [configuring parameters](#configuring-parameters).
+1. In the **AWS Lambda \> Functions \> Application Settings** panel, configure the parameters as described in [configuring parameters](#configuringparameters) section below.
```

Comment:
```suggestion
1. In the **AWS Lambda \> Functions \> Application Settings** panel, configure the parameters as described in [configuring parameters](/docs/send-data/collect-from-other-data-sources/autosubscribe-arn-destination/#configuringparameters).
```

Reply:
try the full path and see if the build still rejects you.

Reply:
Hi Stacy, the cause of the build failure was due to something in the github actions.